src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSPackageMetadataTransformer.java

```diff
@@ -215,34 +216,34 @@ private ViewModel generateMetadataView(
     return dependencies.build();
   }
 
-  private boolean hasLongrunning(ApiModel model, ProductConfig productConfig) {
-    for (InterfaceModel apiInterface : model.getInterfaces()) {
-      if (productConfig.getInterfaceConfig(apiInterface).hasLongRunningOperations()) {
-        return true;
-      }
+  private List<PackageDependencyView> generateDevDependencies(
+      ApiModel model, ProductConfig productConfig) {
+    ImmutableList.Builder<PackageDependencyView> dependencies = ImmutableList.builder();
+    dependencies.add(PackageDependencyView.create("mocha", VersionBound.create("3.2.0", "")));
```

Comment:
hey, I hate this: this feels very wrong to have version numbers hardwired in the code, even though it's Node-specific class.